docs/helpers.rst

```diff
@@ -423,11 +423,12 @@ Example
 ``django_assert_num_queries``
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-.. py:function:: django_assert_num_queries(num, connection=None, info=None)
+.. py:function:: django_assert_num_queries(num, connection=None, info=None, using=None)
 
   :param num: expected number of queries
   :param connection: optional non-default DB connection
   :param str info: optional info message to display on failure
+  :param str using: optional the non-default DB connection name
```

Comment:
"default" would work, right? No need to say "non-default".

```suggestion
  :param connection: optional database connection
  :param str info: optional info message to display on failure
  :param str using: optional database connection name
```

Reply:
I wanted to keep the changes to a minimal and avoid changing things i also disagreed with, but it’s been like this for a long time; going against the grain is always rougher!

Reply:
LOL You’re a member. lol my bad!